tests/unit/forklift/test_legacy.py

```diff
@@ -1081,8 +1080,8 @@ def test_fails_with_admin_flag_set(self, pyramid_config, db_request):
         resp = excinfo.value
 
         assert resp.status_code == 403
-        assert resp.status == ("403 New Project Registration Temporarily "
-                               "Disabled See "
+        assert resp.status == ("403 New Project Registration temporarily "
```

Comment:
I think we should sentence-case this entire line.

Reply:
Done.